RingPublishingGDPR/src/main/java/com/ringpublishing/gdpr/internal/android/Preferences.java

```diff
@@ -88,6 +88,11 @@ public int getInt(String key)
         return preferences.getInt(key, 0);
     }
 
+    public boolean containsInt(String key)
```

Comment:
czy to celowo ma byc publiczne ? nie ma opisu fun

Reply:
to jest publiczne ale w pakiecie internal użyawne tylko wewnętrzenie w naszych klasach. 
Nie jest to w publicznym interfejcie ringPublishingGDPR